app/controllers/concerns/reports/file_responder.rb

```diff
@@ -16,7 +16,11 @@ def render_or_send_by_mail args
   private
 
     def send_report_by_email? collection
-      collection.unscope(:group).count > SEND_REPORT_EMAIL_AFTER_COUNT
+      collection.unscope(
+        :group, :order, :select, :having, :limit, :offset
+      ).select(
+        "#{collection.model.quoted_table_name}.id"
```

Comment:
Le pondría algo así:

```suggestion
        "#{collection.model.quoted_table_name}.#{collection.model.qcn 'id'}"
```

Ya que estamos.